cibuildwheel/linux.py

```diff
@@ -77,8 +74,7 @@ def build(project_dir, package_name, output_dir, test_command, test_requires, be
             # Install packages and test
             for PYBIN in {pybin_paths}; do
                 # Install the wheel we just built
-                "$PYBIN/pip" install {package_name} \
-                    --upgrade --force-reinstall --no-deps --no-index -f /output
+                "$PYBIN/pip" install $("$PYBIN/python" -c "{filter_wheels}" /tmp/linux_wheels/*.whl)
```

Comment:
Yikes! you're absolutely right, I meant to point to the delocated wheel. See my [previous comment](https://github.com/joerick/cibuildwheel/pull/18#issuecomment-313952707) for why I made this change though.